tests/RobotFramework/tests/cumulocity/self-update/tedge_self_update.robot

```diff
@@ -74,6 +73,25 @@ Refreshes mosquitto bridge configuration
     # Mosquitto should be restarted with new bridge
     Execute Command    cmd=sh -c '[ $(journalctl -u mosquitto | grep -c "Loading config file /etc/tedge/mosquitto-conf/c8y-bridge.conf") = 2 ]'
 
+Update tedge version from base to current using Cumulocity
+    ${arch}=    Get Debian Architecture
+    Execute Command    mkdir -p /setup/base-version
+    Transfer To Device    ${CURDIR}/base-version/*${arch}.deb    /setup/base-version/
```

Comment:
It would be good to add a comment on how these packages have been built, notably in case one needs to support another architecture.

Reply:
Will do. It also reminded me that I haven't checked in the dummy packages for other supported platforms.